account/tests.py

```diff
@@ -2,39 +2,62 @@
 import re
 from account.models import validate_gov_id, validate_phone, validate_pin_code
 from django.core.exceptions import ValidationError
+from account.factories import CustomUserFactory
+import django
 
+class TestCustomUserValidators(django.test.TestCase):
+    
+    def test_valid_phone_number(self):
+        user = CustomUserFactory.create(phone='3336647')
+        
+        self.assertTrue(user.phone.isdigit())
+        self.assertEqual(len(user.phone), 7)
 
-class TestCustomUserValidators(unittest.TestCase):
-
-    #test for phone validation
-     def test_phone_validator(self):
+    def test_invalid_phone_number(self):
+        user = CustomUserFactory.create(phone='589929a')
+        with self.assertRaises(ValidationError):
+            validate_phone(user.phone)
```

Comment:
You should not use internal function by yourself, they should be called automatically